dist/gmusic.js

```diff
@@ -207,6 +248,15 @@ proto.playback = {
       return GMusic.Playback.NO_SHUFFLE;
     }
   },
+  setShuffle: function (mode) {
+    if (mode === GMusic.Playback.NO_SHUFFLE || mode === GMusic.Playback.ALL_SHUFFLE) {
+      while (this.playback.getShuffle() !== mode) {
+        this.playback.toggleShuffle();
+      }
+    } else {
+      console.warn('Invalid shuffle mode supplied');
```

Comment:
We shouldn't warn, this should be a hard failure via an error. Please either use `assert` or `throw new Error`. We should prob use `assert` as it's used in other locations:

https://github.com/gmusic-utils/gmusic.js/blob/4.2.0/lib/main.js#L105

```js
assert(GMusic.Playback.SHUFFLE_MODES.indexOf(mode) !== -1, 'Invalid shuffle mode supplied');
// while ...
```